xonsh/environ.py

```diff
@@ -2250,23 +2249,41 @@ def foreign_env_fixes(ctx):
         del ctx["PROMPT"]
 
 
+class _RcPath(str):
+    """A class used exclusively to know which entry was added temporarily to
+    sys.path while loading rc files.
+    """
+
+    pass
+
+
 def xonsh_script_run_control(filename, ctx, env, execer=None, login=True):
     """Loads a xonsh file and applies it as a run control."""
+
```

Comment:
Please remove all of the extra empty lines

Reply:
Done